en/docs/extended_topics/implement_data_loader.md

```diff
@@ -26,14 +26,14 @@ As an example, the data format that Mini Dataloader supported is : A plain text
 
 This article will take Mini Dataloader as an example to explain the key points of implementing customized DataLoader. 
 
-# The composition of Dataloader
+# The Composition of Dataloader
 
 A complete Dataloader generally includes two types of Op: 
 
-- Data Reader: Which is responsible for loading the data in file system to the input stream of memory and setting the data to the Op's output. 
-- Data Decoder: The Data Decoder decodes and outputs the data in Data Reader Op. 
+- Data Reader: It is responsible for taking data from the file and loading it into the input flow in memory. Then ultimately setting the data to the output of the Op. It can be divided into Loader and Parser where Loader is responsible for reading raw data from the file system and Parser is responsible for organizing the raw data into the output of a Data Reader Op.
```

Comment:
```suggestion
- Data Reader: It is responsible for taking data from the file and loading it into the input stream in memory. Then setting the data to the output of the Op. It can be divided into Loader and Parser where Loader is responsible for reading raw data from the file system and Parser is responsible for organizing the raw data into the output of a Data Reader Op.
```